libcontainer/process_linux.go

```diff
@@ -420,7 +420,24 @@ func (p *initProcess) terminate() error {
 	if _, werr := p.wait(); err == nil {
 		err = werr
 	}
-	return err
+	return ignoreTerminateErrors(err)
+}
+
+// ignoreTerminateErrors returns nil if the given err matches an error known
+// to indicate that the terminate occurred successfully or err was nil, otherwise
+// err is returned unaltered.
+func ignoreTerminateErrors(err error) error {
+	if err == nil {
+		return nil
+	}
+
+	// TODO(steve): Update these to none string checks if the runtime exports them.
+	switch err.Error() {
+	case "os: process already finished", "exec: Wait was already called":
```

Comment:
NACK. That's just not a good idea. Can you explain why you need the errors to be filtered? "Just silently ignore them inside a library" smells bad to me.